src/main/java/com/mark0wka/entity/EntityObject.java

```diff
@@ -0,0 +1,41 @@
+package com.mark0wka.entity;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.util.HashSet;
+import java.util.Set;
+
+import static javax.persistence.CascadeType.ALL;
+
+@javax.persistence.Entity
+@Table(name = "object_list")
+@Data
+@NoArgsConstructor
+public class EntityObject {
```

Comment:
I think you need to extend your model with parent_id field to store it like:
Cinema
   Hall1
    seat 1,1
    seat 1.2
   Hall2
    seat 2.1
    seat 2.2

Film catalog
   film 1
   film 2
It can help you with navigation from object to object during search